lib/ProductOpener/Products.pm

```diff
@@ -1303,15 +1362,15 @@ sub store_product ($user_id, $product_ref, $comment) {
 				"moving product images",
 				{
 					source => "$BASE_DIRS{PRODUCTS_IMAGES}/$old_path",
-					destination => "$new_www_root/images/products/$path"
+					destination => "$www_root/images/products/$path"
```

Comment:
In this whole sections you should not use $www_root and $data_root any more…
Please use `%BASE_DIRS` values …